Manifests/ManifestsApple/com.apple.proxy.http.global.plist

```diff
@@ -273,8 +278,26 @@
 			<string>string</string>
 		</dict>
 		<dict>
+			<key>pfm_conditionals</key>
+			<array>
+				<dict>
+					<key>pfm_require</key>
+					<string>always</string>
+					<key>pfm_target_conditions</key>
+					<array>
+						<dict>
+							<key>pfm_range_list</key>
+							<array>
+								<string>Auto</string>
+							</array>
+							<key>pfm_target</key>
+							<string>ProxyType</string>
+						</dict>
+					</array>
+				</dict>
+			</array>
 			<key>pfm_description</key>
-			<string>The URL of the PAC file that defines the proxy configuration.</string>
+			<string>The URL of the PAC file that defines the proxy configuration. Beginning with iOS 13 and macOS 10.15, URLs must be prefixed by http:// or https://.</string>
```

Comment:
While not in the documentation, to help enforce proper formatting, can we add a `pfm_format` key?

```
<key>pfm_format</key>
<string>^https?://.*</string>
```

Reply:
I was hesitant to add this because the format requirement only came into force two versions ago, but if you know from your experience that macOS and iOS have always accepted PAC files via HTTP only, then let's do it.

Reply:
I mean, Apple's docs say http & https, so I'd say let's put in the formatting.

Reply:
You got it.